canonical/ubuntu-pro-client
# Add fix plan API endpoint

## Why is this needed?
We are adding support to two new endpoints:

```
u.pro.security.fix.cve.plan.v1
u.pro.security.fix.usn.plan.v1
```

Those endpoints detail all of the necessary steps we need to actually fix a CVE/USN without executing any of those steps.
This PR is the first step required for us to implement the execute part of the API while also modifying the CLI command to use the endpoints as well

## Test Steps
Run all of the new integration tests and certify that the API endpoint is consistent with the result we see for the CLI output


## Checklist


## Does this PR require extra reviews?
